x-pack/plugins/security_solution/public/management/components/page_overlay/page_overlay.tsx

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, {
+  memo,
+  ReactNode,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+  CSSProperties,
+} from 'react';
+import styled, { createGlobalStyle } from 'styled-components';
+import { EuiFocusTrap, EuiPortal } from '@elastic/eui';
+import classnames from 'classnames';
+import { useLocation } from 'react-router-dom';
+import { EuiPortalProps } from '@elastic/eui/src/components/portal/portal';
+import { EuiTheme } from '@kbn/kibana-react-plugin/common';
+import { TIMELINE_OVERRIDES_CSS_STYLESHEET } from '../../../common/components/page';
+import {
+  SELECTOR_TIMELINE_IS_VISIBLE_CSS_CLASS_NAME,
+  TIMELINE_EUI_THEME_ZINDEX_LEVEL,
+} from '../../../timelines/components/timeline/styles';
+import { useIsMounted } from '../../hooks/use_is_mounted';
+
+const OverlayRootContainer = styled.div`
+  border: none;
+
+  display: block;
+  position: fixed;
+  overflow: hidden;
+
+  top: calc((${({ theme: { eui } }) => eui.euiHeaderHeightCompensation} * 2));
+  bottom: 0;
```

Comment:
Yeah, that (maybe) is true today. I remember having to do both to accommodate some quirk of a specific browser (don't recall which).  Since this does not hurt, I'm going to lave it.